exchangelib/fields.py

```diff
@@ -529,31 +531,44 @@ def to_xml(self, value, version):
         return set_xml_value(field_elem, base64.b64encode(value).decode('ascii'), version=version)
 
 
-class DateField(FieldURIField):
-    value_cls = EWSDate
-
+class GenericDateField(FieldURIField):
```

Comment:
a docstring for this class might be helpful to add!

Reply:
Good idea